ts/store/actions/wallet/wallets.ts

```diff
@@ -125,6 +130,15 @@ export const runStartOrResumeAddCreditCardSaga = createStandardAction(
   "RUN_ADD_CREDIT_CARD_SAGA"
 )<StartOrResumeAddCreditCardSagaPayload>();
 
+/**
+ * search for user's bancomat pan
+ */
+export const addWalletBancomatPan = createAsyncAction(
+  "WALLET_BANCOMAT_ADD_PAN_REQUEST",
+  "WALLET_BANCOMAT_ADD_PAN_SUCCESS",
+  "WALLET_BANCOMAT_ADD_PAN_FAILURE"
+)<Card, Card, Error>();
```

Comment:
```suggestion
)<Card, WalletV2, Error>();
```
This should return a `WalletV2`, right?

Reply:
Yes it should!
Since these types are defined as totally optional I didn't have any type guard!
Fixed within https://github.com/pagopa/io-app/pull/2286/commits/c1b0f92ada512af8767c69e70c72bf345d0ae398

Reply:
@fabriziofff to test these change you need the last commit pushed on io-dev-api-server